asab/library/service.py

```diff
@@ -386,12 +386,13 @@ async def export(self, path="/", tenant=None, remove_path=False) -> typing.IO:
 		return fileobj
 
 
-	async def subscribe(self, paths):
+	async def subscribe(self, paths) -> None:
 		"""
 		It subscribes to the changes in the library
 
 		:param paths: A list of absolute paths to subscribe to
 		"""
+		assert isinstance(paths, list), "The 'paths' parameter in 'LibraryService.subscribe' method must be a list."
```

Comment:
You can also be nice to the user and if the `paths` is a string, you do `paths = [paths]` ...